From 6df202e14c60f495b8acf32fc9271b37aff65bfe Mon Sep 17 00:00:00 2001
From: Oswald Buddenhagen <ossi@kde.org>
Date: Sat, 13 Jun 2009 14:31:52 +0200
Subject: [PATCH 1/5] don't suppress error messages from git availability check

otherwise there is no feedback at all.
as it happens, this also fixes a bashism
---
 maint/version.sh |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/maint/version.sh b/maint/version.sh
index 4df9489..933c14e 100755
--- a/maint/version.sh
+++ b/maint/version.sh
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-git --version &>/dev/null || exit
+git --version >/dev/null || exit
 
 curr_dir=$(pwd)
 
-- 
1.6.3.1

